boreas/boreas_io.c

```diff
@@ -185,7 +185,10 @@ get_host_from_queue (kb_t alive_hosts_kb, gboolean *alive_deteciton_finished)
 
               num_not_scanned_hosts =
                 get_alive_hosts_count () - get_max_scan_hosts ();
-              send_limit_msg (num_not_scanned_hosts);
+              if (0 != num_not_scanned_hosts)
```

Comment:
I would prefer to do the check like this              
`if (num_not_scanned_hosts > 0)` 
because that way it is not dependent on max_scan_host_reached()

Reply:
But the message only has to be sent when `max_scan_host_reached()` returns true. So its dependent on it on purpose.